makefiles/Makefile.docs-compass

```diff
@@ -37,24 +37,29 @@ publish: migrate-assets ## Builds this branch's publishable HTML and other artif
 
 next-gen-html:
 	# snooty parse and then build-front-end
-	curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-compass.yaml > ${REPO_DIR}/published-branches.yaml
-	@echo ${SNOOTY_DB_PWD} | snooty build ${REPO_DIR} "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true"; \
+	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true"; \
 	if [ $$? -eq 1 ]; then \
 		exit 1; \
 	else \
 		exit 0; \
 	fi
-	cp -r ${REPO_DIR}/../../snooty ${REPO_DIR};
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR}; 
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty; 
```

Comment:
make sure this runs sequentially.

Reply:
It is my understanding that makefile always runs commands sequentially unless you've explicitly configured it otherwise.

I can try to find documentation on this from GNU, but my understanding is that make only ever runs one line at a time and always waits for the previous line to finish before running the next one. 

This makes sense if you consider that make has to wait because if the current line fails the target has to fail without starting the next line.


I also checked in with Andrew with this to confirm makefiles are always sequential